runtime/cudaq/qis/execution_manager.h

```diff
@@ -31,6 +31,35 @@ struct QuditInfo {
       : levels(_levels), id(_id) {}
 };
 
+bool __nvqpp__MeasureResultBoolConversion(int);
+
+#ifdef CUDAQ_LIBRARY_MODE
+
+/// @brief In library mode, we model the return type of 
```

Comment:
I think we should do that independent on the mode. Did you only do the change for library mode since applying that change everywhere requires change to the bridge?

Reply:
Yea this will require fixes in the ASTBridge that I was not prepared to do here. Might need Eric's help.